cmd/completion.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(completionCmd)
+}
+
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate command line completion script",
+	Long: fmt.Sprintf(`To load completions:
+
+Bash:
+
+$ source <(%[1]s completion bash)
+
+# To load completions for each session, execute once:
+Linux:
+  $ %[1]s completion bash > /etc/bash_completion.d/%[1]s
+MacOS:
+  $ %[1]s_osx completion bash > /usr/local/etc/bash_completion.d/%[1]s_osx
+
+Zsh:
+
+# If shell completion is not already enabled in your environment you will need
+# to enable it.  You can execute the following once:
+
+$ echo "autoload -U compinit; compinit" >> ~/.zshrc
+
+# To load completions for each session, execute once:
+$ %[1]s completion zsh > "${fpath[1]}/_%[1]s"
```

Comment:
$fpath[1] seems to point to `.oh-my-zsh/plugins/colored-man-pages` for me, I'm assuming that's not where the completions should end up.

Reply:
Added an extra comment in the help text on this. Cant come up with a simple standardized way here.